src/ray/object_manager/pull_manager.cc

```diff
@@ -343,6 +343,8 @@ void PullManager::UpdatePullsBasedOnAvailableMemory(int64_t num_bytes_available)
 
   // Call the cancellation callbacks outside of the lock.
   for (const auto &obj_id : object_ids_to_cancel) {
+    RAY_LOG(INFO) << "Not enough memory to create requested object " << obj_id
```

Comment:
Can you remove this INFO message? I think this message will be spammy.

Reply:
This was needed because the info log in `AbortCreate()` was removed. I reverted this change and may send out the info log change separately.